src/styles/_homepage.scss

```diff
@@ -0,0 +1,74 @@
+// Dots background pattern
+.homepage--dots {
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 16 16' style='enable-background:new 0 0 16 16;' xml:space='preserve'%3e%3crect width='16' fill='none' height='16'/%3e%3crect x='0' y='0' fill='white' width='1' height='1'/%3e%3c/svg%3e");
+  background-size: 16px;
+  opacity: 0.35;
+}
+
+//make sure content is above dots
+.container--homepage div[class*='Main-module--main'],
+.container--homepage div[class*='Callout-module--grid'],
+.container--homepage div[class*='NextPrevious-module--wrapper'] {
+  position: relative;
+  z-index: 2;
+}
+
+.homepage--header {
+  height: 0;
+  overflow: hidden;
+  padding-top: 100%;
+  position: relative;
+
+  @include carbon--breakpoint('md') {
+    padding-top: 50%;
+  }
+
+  // Match max height at lg breakpoint
+  @include carbon--breakpoint('lg') {
+    padding: 0;
+    height: 560px;
+  }
+}
+
+.homepage--header .bx--grid,
+.homepage--header .bx--row {
+  height: 100%;
+}
+
+// homepage tile at top
+.homepage--tile-header {
+  display: flex;
+  align-items: flex-end;
+  margin-top: -50%;
+  padding-left: 0;
+  padding-right: 0;
+
+  @include carbon--breakpoint('md') {
+    padding-left: $spacing-05;
+    padding-right: $spacing-05;
+  }
+
+  @include carbon--breakpoint('lg') {
+    margin-top: 0;
+  }
+
+  .bx--aspect-ratio--2x1 {
+    padding-bottom: 100%;
+
+    @include carbon--breakpoint('md') {
+      padding-bottom: 50%;
+    }
+  }
+}
```

Comment:
@vpicone this was the style override I was doing to get the 1-1 ratio for mobile. agree we need a better way

Reply:
ahhh I see, yeah I think that'd crop the developer one